src/yggdrasil/tun.go

```diff
@@ -42,11 +44,34 @@ func getSupportedMTU(mtu int) int {
 func (tun *tunAdapter) init(core *Core, send chan<- []byte, recv <-chan []byte) {
 	tun.Adapter.init(core, send, recv)
 	tun.icmpv6.init(tun)
+	go func() {
+		for {
+			select {
+			case e := <-tun.reconfigure:
+				tun.core.configMutex.RLock()
+				updated := tun.core.config.IfName != tun.core.configOld.IfName ||
+					tun.core.config.IfTAPMode != tun.core.configOld.IfTAPMode ||
+					tun.core.config.IfMTU != tun.core.configOld.IfMTU
+				tun.core.configMutex.RUnlock()
+				if updated {
+					e <- nil
```

Comment:
We should probably print some kind of log message or something, so people at least know that tun/tap changes aren't supported if they attempt it, to (hopefully) get them to restart the process.

Reply:
Agreed - I've updated this to print a message.